windows/ReactNativeWebView/ReactWebView2.cpp

```diff
@@ -160,10 +160,14 @@ namespace winrt::ReactNativeWebView::implementation {
             });
     }
 
-    void ReactWebView2::SetMessagingEnabled(bool enabled) {
+    void ReactWebView2::MessagingEnabled(bool enabled) {
         m_messagingEnabled = enabled;
     }
 
+    bool ReactWebView2::MessagingEnabled() {
+        return m_messagingEnabled;
+    }
```

Comment:
if you take the other feedback, update here:
```suggestion
    void ReactWebView2::MessagingEnabled(bool enabled) noexcept {
        m_messagingEnabled = enabled;
    }

    bool ReactWebView2::MessagingEnabled() const noexcept {
        return m_messagingEnabled;
    }
```